Backend/DynamicQR/Api.Tests/Mappers/QrCodesMappersTests.cs

```diff
@@ -0,0 +1,203 @@
+﻿using DynamicQR.Api.Mappers;
+using FluentAssertions;
+using System.Diagnostics.CodeAnalysis;
+using System.Drawing;
+
+namespace Api.Tests.Mappers;
+
+[ExcludeFromCodeCoverage]
```

Comment:
@jensdenbraber waarom moet dit attribute [ExcludeFromCodeCoverage] toegevoegd worden binnen een Test project? Ik zou verwachten dat een testproject als geheel sowieso al niet meegenomen wordt in de code coverage. Of heb ik dat verkeerd?